transition-scripts/values/values-gold-eb75ad-dev-standby.yaml

```diff
@@ -25,7 +25,7 @@ patroni:
     size: 10Gi
 
   podDisruptionBudget:
-    enabled: true
+    enabled: false
```

Comment:
Why is pdb here disabled?

Reply:
because we set the patroni pod count to 1; if pdb is enabled, the pod can not be re-created as `minAvailable is set to 1` by default.